ansible/roles/csm.rr.kyverno_policy/files/apply_topology_policy_restart.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python3
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+"""
+Apply RR ClusterPolicy and do rollout restart of the critical services defined
+in RR static ConfigMap.
+"""
+
+import yaml
+import json
+import subprocess
+import sys
+import tempfile
+import os
+
+
+def load_configmap(name, namespace):
```

Comment:
I am not doing a formal review of this yet, but just wanted to point out that doing this Kubernetes stuff inside the CFS pod using Ansible will make your life easier. The more you can do in Ansible, the fewer bugs you'll run into due to weirdness on the nodes, and the faster it will run.